### DISCUSSION TITLE
Redis

### DISCUSSION THREAD

**Discussion by juliacollins** [nsidc/tech-radar#39] [Tools]
Web database. Currently used by Dataset Catalog Services.

**Comment by MattF-NSIDC** [comment]
>Web database

I think it's important to differentiate Redis (**Re**mote **Di**ctionary **S**erver) from SQL databases (e.g. PostgreSQL) and document databases (e.g. MongoDB). Redis is an in-memory key-value store. Keys are strings, and values can be [various data structures](https://redis.com/redis-enterprise/data-structures/) like strings, sets, lists, hashes. Redis only supports query by key.
